client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx

```diff
@@ -0,0 +1,555 @@
+                                } else {
+                                    deletionNote = (
+                                        <>
+                                            Note that {testsToDelete.length}{' '}
+                                            {testsToDelete.length === 1
+                                                ? 'test differs'
+                                                : 'tests differ'}{' '}
+                                            between the old and new versions and
+                                            cannot be automatically copied.
+                                        </>
+                                    );
+                                }
+
+                                return (
+                                    <Alert
+                                        variant={
+                                            testsToDelete.length
+                                                ? 'danger'
+                                                : 'info'
+                                        }
+                                    >
+                                        Found {allTestResults.length} partial or
+                                        completed test{' '}
+                                        {allTestResults.length === 1
+                                            ? 'result'
+                                            : 'results'}{' '}
+                                        for{' '}
+                                        {testers.length > 1
+                                            ? 'testers'
+                                            : 'tester'}{' '}
+                                        {toSentence(testers)}. {deletionNote}
+                                    </Alert>
+                                );
+                            })()}
+                        </div>
+                    </Modal.Body>
+                    <Modal.Footer className="test-plan-updater-footer">
+                        <div className="submit-buttons-row">
+                            {testsToDelete?.length > 0 && (
+                                <Form.Check
+                                    type="checkbox"
+                                    className="backup-checkbox"
+                                >
+                                    <Form.Check.Label>
+                                        <Form.Check.Input
+                                            type="checkbox"
+                                            onChange={() => {
+                                                setBackupChecked(
+                                                    !backupChecked
+                                                );
+                                            }}
+                                            checked={backupChecked}
+                                        />
+                                        Backup Test Plan before Update
```

Comment:
What would you think about changing this to "Retain old test plan"?

Or if the copy is pretty-much decided already, could you change this to "Back up test plan before update"? I do prefer the word "retain" if that's possible.

Reply:
I have no preference. @s3ththompson @isaacdurazo what do you think is the preferred wording?

Reply:
I do think the existing test will work, thanks for updating it.